internal/handlers/pg/msg_datasize.go

```diff
@@ -47,17 +46,17 @@ func (h *Handler) MsgDataSize(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg
 
 	common.Ignored(document, h.L, "estimate")
 
-	m := document.Map()
-	target, ok := m["dataSize"].(string)
-	if !ok {
-		return nil, lazyerrors.New("no target collection")
-	}
-	targets := strings.Split(target, ".")
-	if len(targets) != 2 {
-		return nil, lazyerrors.New("target collection must be like: 'database.collection'")
+	command := document.Command()
+
+	target, err := common.GetRequiredParam[string](document, command)
+	if err != nil {
+		return nil, lazyerrors.Error(err)
```

Comment:
Could the error message tested, or at least have skipped tests for missing and wrong type please?

Reply:
Added test for an object passed instead of a string.